ext/templates/puppetdb-ssl-setup.erb

```diff
@@ -226,7 +226,7 @@ else
   agent_confdir=`puppet agent --configprint confdir`
   agent_vardir=`puppet agent --configprint vardir`
 
-  if [ -d "/etc/puppetlabs/puppetdb" ] ; then
+  if id -u pe-puppetdb > /dev/null 2>&1; then
```

Comment:
Looks like `id` is posix (IEEE 1003.1), `getent` is mainly from glibc, fwiw.

* http://pubs.opengroup.org/onlinepubs/9699919799/utilities/id.html
* http://linux.die.net/man/1/getent